addon/authenticators/base.js

```diff
@@ -7,16 +7,17 @@ const { RSVP, Evented, Object: EmberObject } = Ember;
   implementing custom authenticators and must not be used directly.__
 
   The authenticator authenticates the session. The actual mechanism used to do
-  this might e.g. be posting a set of credentials to a server and in exchange
-  retrieving an access token, initiating authentication against an external
-  provider like Facebook etc. and depends on the specific authenticator. Any
-  data that the authenticator receives upon successful authentication and
-  resolves with from the
+  this might, e.g., post a set of credentials to a server and in exchange
+  retrieve an access token, initiating authentication against an external
+  provider like Facebook, etc. The details depend on the specific authenticator. Upon
+  successful authenticateion, any
+  data that the authenticator receives and
+  resolves via the promise returned from the
```

Comment:
This seems to be formatted a bit strange. In general, for all non-code text in the codebase we're adhering to a 80 chars limit per line.